src/components/SearchBar.js

```diff
@@ -1,14 +1,19 @@
-import React, { Component } from 'react'
-import ResultsList from './containers/ResultsList'
-import search from '../js/search'
```

Comment:
Ok, in the future, if you don't know how to improve something, just leave it for now, and we can pair program the changes together if you like. So please revert the code to a functional state, and I will approve once you make the additional changes requested.